index.js

```diff
@@ -419,6 +419,52 @@
     return f(y, x);
   });
 
+  //# lift :: Applicative f => (a -> b) -> f a -> f b
+  //.
+  //. Takes a unary function and promotes it to a function which operates on
+  //. a functor.
+  //.
+  //. ```javascript
+  //. > S.lift(R.inc, S.Just(2))
+  //. S.Just(3)
+  //.
+  //. > S.lift(R.inc, S.Nothing())
+  //. S.Nothing()
+  //. ```
+  S.lift = def('lift', [Function, a], R.map);
+
+  //# Applicative f => (a -> b -> c) -> f a -> f b -> f c
+  //.
+  //. Takes a binary function and promotes it to a function which operates on
+  //. two applicative functors.
+  //.
+  //. ```javascript
+  //. > S.lift2(R.add, S.Just(2), S.Just(3))
```

Comment:
Could we add an example using `S.and` in the example to show people how to roll their own typesafe version of `R.both` as discussed in #110?